src/app.css

```diff
@@ -0,0 +1,214 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  width: 100%;
+  height: 80px;
+  display: flex;
+  flex-direction: row;
+  background-color: #f1f1f1;
+}
+
+.nav-main {
+  display: flex;
+  width: 95%;
+  margin: 0 auto;
+}
+
+.logo {
+  width: 20%;
+}
+
+.logo img {
+  width: 85px;
+  height: auto;
+}
+
+.menu {
+  width: 55%;
+  display: inline-block;
+  text-align: center;
+}
+
+.menu ul li {
+  display: inline-block;
+  margin: 30px 15px;
+  vertical-align: middle;
+}
+
+.menu ul li a, .m-menu li a {
+  font-size: 15px;
+  color: #161940;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.menu ul li a:hover {
+  border-bottom: 1px solid #161940;
+}
+
+.cta {
+  width: 25%;
+  margin: auto 0;
+  text-align: right;
+}
+
+.cta-login {
+  background-color: #4b4d6c;
+  color: #fff;
+  font-weight: 600;
+  padding: 10px 20px;
+  margin: 10px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.cta-reg {
+  background-color: #40447c;
+  color: #fff;
+  font-weight: 600;
+  padding: 10px 20px;
+  margin: 10px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.cta-reg:hover, .cta-login:hover {
+  color: #fff;
+  box-shadow: 1px 2px 8px 4px rgba(164, 164, 208, 0.52);
+}
+
+.fixed-top {
+  position: sticky;
+  top: 0;
+  z-index: 99;
+}
+
+.footer {
+  background-color: #161940;
+  width: 100%;
+}
+
+.ft {
+  width: 90%;
+  margin: 0 auto;
+  padding: 50px 0;
+  display: flex;
+}
+
+.ft-col {
+  width: 33.33%;
+}
+
+.ft-col img {
+  width: 100px;
+}
+
+.ft-col ul {
+  margin: 0 30%;
+}
+
+.ft-col ul li {
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.ft-col ul li a {
+  color: #fff;
+  line-height: 40px;
+  font-size: 15px;
+}
+
+.ft-col ul li a:hover {
+  color: #f1f1f1;
+}
+
+.copyright {
+  padding: 20px 0;
+  border-top: 2px dashed #fff;
+}
+
+.copy {
+  text-align: center;
+  color: #fff;
+  font-size: 13px;
+}
+
+@media only screen and (min-width:900px) {
+  .hambuger {
+    display: none;
+  }
+}
+
+@media only screen and (max-width:768px) {
+  .logo {
+    width: 50%;
+  }
+
+  .logo img {
+    width: 45%;
+  }
+
+  .menu, .cta {
+    display: none;
+  }
+
+  .bar {
+    width: 100%;
+    display: block;
+    border: 1px solid #000;
+    text-align: right;
+    margin: 9px 0;
+  }
+
+  .hambuger {
+    width: 20px;
+    margin: auto 15px;
+  }
+
+  .nav-main {
+    justify-content: space-between;
+  }
+
+  .ft {
+    flex-direction: column;
+  }
+
+  .ft-col {
+    width: 100%;
+  }
+
+  .ft-col ul {
+    margin: 20px;
+  }
+
+  .copy {
+    text-align: left;
+    margin-left: 40px;
+  }
+
+  .mobile-menu {
+    display: block;
+    width: 100%;
+    height: 100vh;
+    position: fixed;
+    background-color: #fff;
+  }
+
+  .m-menu {
+    margin-top: 30px;
+    list-style-type: none;
+    line-height: 40px;
+    text-align: center;
+  }
+
+  .m-cta {
+    display: flex;
+    flex-direction: column;
+    margin: auto 30%;
+    text-align: center;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/BuildForSDG/EduKolab-Frontend/pullRequest?prid=5585842)